llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -910,6 +910,7 @@ SubtargetEmitter::FindWriteResources(const CodeGenSchedRW &SchedWrite,
                                           ProcModel.ModelName);
       }
       ResDef = WR;
+      break;
```

Comment:
Doesn't this make the PrintFatalError dead code?

Reply:
I messed up. Need to take a second look at this.

Reply:
I have addressed this problem